src/processTargets/targets/PositionTarget.ts

```diff
@@ -102,37 +105,23 @@ export default class PositionTarget extends BaseTarget {
   }
 
   private getEditText(text: string) {
-    if (this.isLineDelimiter) {
-      return this.isBefore
-        ? this.linePadding + text + this.insertionDelimiter
-        : this.insertionDelimiter + this.linePadding + text;
-    }
+    const insertionText = this.indentationString + text;
```

Comment:
We don't need the special casing, because `indentationString` is just empty when `!this.isLineDelimiter`